geth/account/proxy_test.go

```diff
@@ -0,0 +1,36 @@
+package account
+
+import (
+	"github.com/ethereum/go-ethereum/accounts"
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+	"github.com/ethereum/go-ethereum/whisper/whisperv5"
+	"github.com/golang/mock/gomock"
+	"github.com/status-im/status-go/geth/common"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestProxy_Base_Success(t *testing.T) {
```

Comment:
Once again I'm personally in trouble with underscores in identifiers, even if they are _only_ for tests.

Reply:
This is test..